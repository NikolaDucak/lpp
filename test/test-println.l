// printing literals
// println must work because testing relies on output
println(5);
println(true);
println(false);
println(nil);
println("strliteral");
