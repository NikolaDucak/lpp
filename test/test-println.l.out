5
true
false
nil
strliteral
